projects/gnomad/src/server/server.js

```diff
@@ -8,6 +8,15 @@ const port = process.env.PORT || 80
 const app = express()
 app.use(compression())
 
+// Redirect HTTP requests to HTTPS
+app.use((request, response, next) => {
+  if (request.get('x-forwarded-proto') === 'http') {
+    response.redirect(307, `https://${request.get('host')}${request.url}`)
```

Comment:
Is it a 301 redirect? https://support.google.com/webmasters/answer/6073543?hl=en, what happens when we don't specify a status code?

Reply:
[307](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307). MDN recommended 307/308 to avoid changing the request method/body. I was thinking that would be necessary for GraphQL requests to the API, but this code won't see those requests. So 301 should work.

Reply:
Should we add a redirect or deny http requests for graphql too?